webapp/src/js/bootstrapper/translator.js

```diff
@@ -58,7 +58,7 @@ const TRANSLATIONS = {
     LOGIN: 'Ingia',
   },
   ne: {
-    FETCH_INFO: ({ count, total }) => eurodigit.to_non_euro.devanagari(`डकुमेन्ट लोड हुँदै  (${count} मध्ये ${total} डकुमेन्ट)…`), // eslint-disable-line max-len
+    FETCH_INFO: ({ count, total }) => eurodigit.to_non_euro.devanagari(`डकुमेन्ट लोड हुँदै  (${total} मध्ये ${count} डकुमेन्ट)…`), // eslint-disable-line max-len
```

Comment:
It might look better if we phrase the sentence in proper Nepali way.
```suggestion
    FETCH_INFO: ({ count, total }) => eurodigit.to_non_euro.devanagari(`(${total} मध्ये ${count}) डकुमेन्ट लोड हुँदै  …`), // eslint-disable-line max-len
```

Reply:
If we phrase it like this, we might not even need the brackets.

Reply:
Let's go with that.